src/backend/distributed/sql/udfs/citus_job_status/latest.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_job_status (
+    job_id bigint DEFAULT NULL,
```

Comment:
I think this argument should be required.

Reply:
Makes sense, I dropped the default value.